pkg/kubeapiserver/authorizer/config.go

```diff
@@ -172,7 +173,7 @@ func LoadAndValidateData(data []byte, requireNonWebhookTypes sets.Set[authzconfi
 	}
 
 	// validate the file and return any error
-	if errors := validation.ValidateAuthorizationConfiguration(nil, authorizationConfiguration,
+	if errors := validation.ValidateAuthorizationConfiguration(authorizationcel.DefaultCompiler(), nil, authorizationConfiguration,
```

Comment:
we've had terrible trouble with cel envs getting locked by something called at init time before flags and feature gates are finalized... this was slightly untangled in https://github.com/kubernetes/kubernetes/pull/125571 (see the `[Move CEL env initialization out of package init()](https://github.com/kubernetes/kubernetes/pull/125571/commits/03d48b76831a3a02d503c3075d818a76afd83cd8)` commit) but I worry this is re-tangling things. The integration test related to this area failing in this PR doesn't give me confidence :-/